frontend/app/utils/date-utils.ts

```diff
@@ -194,6 +194,11 @@ export function toLocaleDateString(date: UTCDate, locale: string) {
   return date.toLocaleDateString(`${locale}-CA`, { year: 'numeric', month: 'long', day: 'numeric' });
 }
 
+export function getYear(dateString: string) {
+  invariant(isValidDateString(dateString) && !isNaN(parseInt(dateString.substring(0, 4), 10)), `Invalid date [${dateString}]`);
+  return dateString.substring(0, 4);
+}
```

Comment:
Consider introducing unit tests to test this function with valid dates and invalid dates (eg. dates in a different format, dates with weird characters, year that is three characters, etc).

Alternatively, we could use the existing `extractDateParts(..)` (in this same file) to get the year from a date string.

Reply:
Missed the `extractDateParts`. Will reuse that instead. Fixed: b41c8079f3f88d4afce1b05534871d24cab7bd5a